csharp/ql/test/library-tests/controlflow/graph/ExtractorOptions.cs

```diff
@@ -0,0 +1 @@
+// semmle-extractor-options: /langversion:8.0
```

Comment:
Could it make sense to have a suite of extractor options files that we can just drop into various qltest files? This file could be called `LanguageVersion8.0.cs` for example.

Reply:
I guess so. Not sure if qltest can handle multiple `// semmle-extractor-options` comments, though.